content/browser/android/content_feature_list.cc

```diff
@@ -42,6 +46,9 @@ const base::Feature* const kFeaturesExposedToJava[] = {
     &features::kWebBluetoothNewPermissionsBackend,
     &features::kWebNfc,
     &features::kDoNotGenerateChromiumA11yTree,
+    #if BUILDFLAG(USE_STARBOARD_MEDIA)
+    &media::kCobaltUsingAndroidOverlay,
```

Comment:
We should try to avoid modifying //content as much as possible,
because changes to this folder a) would imply other embedders
such as Chrome, or Edge, need this code, which is not true and
b) will make future rebases complicated.

Maybe we can define the flag somewhere under //cobalt ?